data_tests/queries

```diff
@@ -0,0 +1,265 @@
+SELECT * FROM "h2o_feet" LIMIT 3'
```

Comment:
Is this whole file the queries from the docs? This file would currently be ignored by the tool, right?